### DISCUSSION TITLE
change name of `next.something` and `next-data`

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [nodejs/nodejs.org#7145] [Ideas]
I don't think these names are 100% right. 

Because 95% of them are server side code, but not specific to nextjs. For example, it could be integrated with remix or fatsify.

So I think we could review the appointment of the rosters and the structure they are in.

**Comment by ovflowd** [comment]
I'm +1 renaming these.

We could call them `engine.XXX` (for `next.XXX`) and `server-data` for (`next-data`)

**Reply by AugustinMauroy** [reply]
What's done quite a bit too, but that's up to the devs, is to put all that in a `lib` directory, for example, which is what next.js offers for setting up prisma.

**Reply by vintageshutter06-svg** [reply]
i am so happy to join you guys here in githup, heard so much about it
